cpp/src/parquet/column_reader_test.cc

```diff
@@ -356,6 +356,34 @@ TEST_F(TestPrimitiveReader, TestReadValuesMissing) {
                ParquetException);
 }
 
+// Repetition level byte length reported in Page but Max Repetition level
+// is zero for the column.
+TEST_F(TestPrimitiveReader, TestRepetitionLvlBytesWithMaxRepetitionZero) {
+  constexpr int batch_size = 4;
+  max_def_level_ = 1;
+  max_rep_level_ = 0;
+  NodePtr type = schema::Int32("a", Repetition::OPTIONAL);
+  const ColumnDescriptor descr(type, max_def_level_, max_rep_level_);
+  const std::array<uint8_t, 21> page_data{0x3,  0x3, 0x7, 0x80, 0x1, 0x4, 0x3,
```

Comment:
it is possble to provide some explanation of where these bytes cape from?

Reply:
sure, in this case i just took them from the example parquet file in the JIRA card, the important bit is the first three bytes. If the first byte isn't skipped then RLE decoding yields [1, 1, 0, 0] instead of the correct [1, 1, 1, 0] for the definition levels. I'll add a comment.

Reply:
yeah documented the important piece would be useful.